modules/final_radiological_review/php/NDB_Menu_Filter_final_radiological_review.class.inc

```diff
@@ -163,104 +167,105 @@ class NDB_Menu_Filter_final_radiological_review extends NDB_Menu_Filter
             ";
 
         $this->columns = array(
-            'c.PSCID as PSCID',
-            'c.CandID as DCCID',
-            's.visit_label as Visit_Label',
-        );
-        if($useProjects) {
+                          'c.PSCID as PSCID',
+                          'c.CandID as DCCID',
+                          's.visit_label as Visit_Label',
+                         );
+        if ($useProjects) {
             $this->columns[] = 'pjct.Name as Project';
         }
 
-        $this->columns = array_merge(
+        $this->columns  = array_merge(
```

Comment:
I think this would be better addressed by adding a newline before the next equal that it's trying to align with rather than changing the spacing here, because the array is long enough that aligning the equals doesn't make sense.